src/components/OptionsList/BaseOptionsList.tsx

```diff
@@ -110,16 +108,6 @@ function BaseOptionsList(
         flattenedData.current = buildFlatSectionArray();
     });
 
-    useEffect(() => {
-        if (isLoading) {
-            return;
-        }
-
-        // Mark the end of the search page load time. This data is collected only for Search page.
-        Timing.end(CONST.TIMING.OPEN_SEARCH);
```

Comment:
Just TBC, you're just removing this metric, not fixing it. Is that the intention here?

Reply:
Yes, the Search page was recently refactored. Before the BaseOptionsList was used for search, but thats not longer the case. Thus the usage here can be cleaned up